jetty-server/src/main/java/org/eclipse/jetty/server/Server.java

```diff
@@ -78,7 +78,7 @@
     private final ThreadPool _threadPool;
     private final List<Connector> _connectors = new CopyOnWriteArrayList<>();
     private SessionIdManager _sessionIdManager;
-    private boolean _stopAtShutdown;
+    private boolean _stopAtShutdown = true;
```

Comment:
This will be a regression for embedded Jetty users that expect it to be false.

The true setting is only relevant for users of jetty-home / jetty-maven-plugin.
This default should be declared in the `server.mod` and code for jetty-maven-plugin.

Reply:
I really don't like the default being different .... but I understand that would be a change for embedded users. I'll make the default back to `true` for xml.